test/eth.accounts.signTransaction.js

```diff
@@ -27,6 +27,7 @@ var tests = [
             nonce: 0,
             gasPrice: "20000000000",
             gas: 21000,
+            gasLimit: 21000,
```

Comment:
If we default `gasLimit` to `gas`, then these changes should be reverted

Reply:
revert this and a test that if gasLimit is passed that it is used and not the default